src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -48,147 +46,95 @@
+        ProTerritoryValueUtils.findTerritoryValues(player, new ArrayList<Territory>(), clearedTerritories);
+    determineTerritoriesThatCanBeHeld(attackOptions, territoryValueMap);
+    prioritizeAttackOptions(player, attackOptions);
+    removeTerritoriesThatArentWorthAttacking(attackOptions);
 
     // Determine which territories to attack
-    determineTerritoriesToAttack(attackMap, unitAttackMap, prioritizedTerritories, transportMapList,
-        transportAttackMap, bombardMap);
+    determineTerritoriesToAttack(attackOptions);
 
     // Determine which territories can be held and remove any that aren't worth attacking
-    territoriesToAttack = new ArrayList<Territory>();
+    clearedTerritories = new ArrayList<Territory>();
     final Set<Territory> possibleTransportTerritories = new HashSet<Territory>();
-    for (final ProTerritory patd : prioritizedTerritories) {
-      territoriesToAttack.add(patd.getTerritory());
+    for (final ProTerritory patd : attackOptions) {
+      clearedTerritories.add(patd.getTerritory());
       if (!patd.getAmphibAttackMap().isEmpty()) {
         possibleTransportTerritories.addAll(data.getMap().getNeighbors(patd.getTerritory(), Matches.TerritoryIsWater));
       }
     }
-    enemyAttackOptions =
-        attackOptionsUtils.findEnemyAttackOptions(player, territoriesToAttack, new ArrayList<Territory>(
-            possibleTransportTerritories));
+    territoryManager.populateEnemyAttackOptions(clearedTerritories, new ArrayList<Territory>(
+        possibleTransportTerritories));
     territoryValueMap =
-        territoryValueUtils.findTerritoryValues(player, minCostPerHitPoint, new ArrayList<Territory>(),
-            territoriesToAttack);
-    determineTerritoriesThatCanBeHeld(prioritizedTerritories, attackMap, enemyAttackOptions, territoryValueMap);
-    removeTerritoriesThatArentWorthAttacking(prioritizedTerritories, enemyAttackOptions);
+        ProTerritoryValueUtils.findTerritoryValues(player, new ArrayList<Territory>(), clearedTerritories);
+    determineTerritoriesThatCanBeHeld(attackOptions, territoryValueMap);
+    removeTerritoriesThatArentWorthAttacking(attackOptions);
 
     // Determine how many units to attack each territory with
-    final List<Unit> alreadyMovedUnits =
-        moveOneDefenderToLandTerritoriesBorderingEnemy(attackMap, unitAttackMap, prioritizedTerritories,
-            myUnitTerritories);
-    determineUnitsToAttackWith(attackMap, enemyAttackOptions, unitAttackMap, prioritizedTerritories, transportMapList,
-        transportAttackMap, bombardMap, alreadyMovedUnits);
+    final List<Unit> alreadyMovedUnits = moveOneDefenderToLandTerritoriesBorderingEnemy(attackOptions);
+    determineUnitsToAttackWith(attackOptions, alreadyMovedUnits);
 
     // Get all transport final territories
     ProMoveUtils.calculateAmphibRoutes(player, new ArrayList<Collection<Unit>>(), new ArrayList<Route>(),
```

Comment:
It appears that only two out of six parameters are actually used in this method call.